lib/puppet/parser/functions/consul_sorted_json.rb

```diff
@@ -6,11 +6,13 @@ class << self
 
     def sorted_generate(obj)
       case obj
-        when Fixnum, Float, TrueClass, FalseClass, NilClass
-          return obj.to_json
+        when NilClass
+          return "null"
```

Comment:
I'd think that generating content having it return the value "null" becomes more problematic IMO.  I'd personally like if something is actually null the generated content is empty "".